sci-libs/gdal/gdal-3.4.2.recipe

```diff
@@ -235,6 +239,7 @@ BUILD()
 {
 	export LDFLAGS="-lnetwork"
 	expatDir=$portPackageLinksDir/lib~libexpat
+	touch config.rpath
```

Comment:
Is this needed, I thought this was fixed upstream?

Reply:
Not sure, I saw it was added in https://github.com/haikuports/haikuports/commit/91a9f05086d4eea546a739fc80969ab4593d17d6. I'll remove it and see what happens.

Reply:
Yeah it was needed there, upon checking I saw mentions that this was fixed upstream. :)